rllib/algorithms/callbacks.py

```diff
@@ -75,13 +75,77 @@ def on_algorithm_init(
         """
         pass
 
+    @OverrideToImplementCustomLogic
+    def on_workers_recreated(
+        self,
+        *,
+        algorithm: "Algorithm",
+        worker_set: "WorkerSet",
+        worker_ids: List[int],
+        is_evaluation: bool,
+        **kwargs,
+    ) -> None:
+        """Callback run after one or more workers have been recreated.
+
+        You can access (and change) the worker(s) in question via the following code
+        snippet inside your custom override of this method:
+
+        Note that any "worker" inside the algorithm's `self.worker` and
+        `self.evaluation_workers` WorkerSets are instances of a subclass of EnvRunner.
+
+        .. code-block:: python
```

Comment:
We don't use code-blocks anymore, we use `testcode` and `testoutput` these days!

Reply:
Oh no! :D Will fix.

Reply:
done